packages/tools/client-debugger/client-debugger/src/index.ts

```diff
@@ -83,3 +85,4 @@ export {
 	getFluidClientDebuggers,
 	initializeFluidClientDebugger,
 } from "./Registry";
+export { DevToolsExtensionLogger } from "./devToolsExtensionLogger";
```

Comment:
We'd prefer to not explicitly call out the fact that this is for our extension (or the fact that it leverages the browser's DevTools panel). Feedback on a descriptive alternative name is welcome.

Reply:
Renamed to `FluidDebuggerLogger`, which makes sense since it also directs the telemetry to the inline-rendered version of the debugger, not just the extension one.